src/containers/Accounts/LoginCallback.js

```diff
@@ -108,9 +114,6 @@ LoginCallback.propTypes = {
   }).isRequired,
   updateUser: PropTypes.func.isRequired,
   updateValidAuth: PropTypes.func.isRequired,
-  intl: PropTypes.shape({
-    formatMessage: PropTypes.func.isRequired,
-  }).isRequired,
 };
 
-export default withCookies(injectIntl(connect(null, mapDispatchToProps)(LoginCallback)));
```

Comment:
`injectIntl` isn't needed anymore?

Reply:
Yes because `FormattedMessage` is used now